fetch_data/soup_data/software/create_software_version_data_observation.py

```diff
@@ -28,6 +29,15 @@ async def create_software_version_observation(
     """Create Software Version Observation"""
 
     info.context["background_tasks"].add_task(update_software_data)
+    return await create_software_version_observation_without_background_task(
+        wikibase_id
+    )
+
+
+async def create_software_version_observation_without_background_task(
```

Comment:
Why is this necessary?

Reply:
Because I couldn't provide a genuine strawberry.Info when calling this method, as I'm calling it outside of a mutation, and mocking one outside of tests seemed bad